CHANGES_NEXT_RELEASE

```diff
@@ -6,3 +6,4 @@
 - [cygnus-ngsi][hardening] Pre-aggregate batches in NGSISTHSink (#1359)
```

Comment:
This PR deals with a purelly internal Cygnus aspect? No impact in documentation?